File: CartQuestAndroid/app/src/main/java/com/amoghghadge/cartquestandroid/data/model/StoreStop.kt
package com.amoghghadge.cartquestandroid.data.model

data class StoreStop(
    val storeId: String,
    val storeName: String,
    val address: String,
    val lat: Double,
    val lng: Double,
    val items: List<AssignedItem>
)

data class AssignedItem(
    val productId: String,
    val name: String,
    val brand: String,
    val price: Double
)

File: CartQuestiOS/CartQuestiOS/Models/StoreStop.swift
import Foundation

struct AssignedItem: Codable, Identifiable {
    var id: String { productId }
    var productId: String
    var name: String
    var brand: String
    var price: Double
}

struct StoreStop: Codable, Identifiable {
    var id: String { storeId }
    var storeId: String
    var storeName: String
    var address: String
    var lat: Double
    var lng: Double
    var items: [AssignedItem]
}
